english/132-days-ago/click-me

```diff
@@ -0,0 +1 @@
+create-your-own-adventure/english/experiment/Dont_open.md
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.